chirp/drivers/retevis_ha1g.py

```diff
@@ -1301,6 +1378,8 @@ def get_features(self):
 
     def process_mmap(self):
         self._memobj = bitwise.parse(MEM_FORMAT, self._mmap)
+        self._dtmf_list = get_dtmf_item_list(self)
+        self._alarm_list = get_alarm_item_list(self)
```

Comment:
These should be methods, called via `self.get_alarm_item_list()` and not functions passed `self` as a parameter.